general_rules/Scenario.tex

```diff
@@ -1,202 +1,122 @@
-\end{enumerate}
-
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%
-% Objects section.
-%
-% Revisited by Mauricio Matamoros for 2015
-%
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\def\NumObjects{30\ }
-\def\NumLocations{20\ }
-\def\NumNames{20\ }
-
 \subsection{Objects}
-\label{rule:scenario_objects}
-Some tests in the RoboCup@Home league involve recognizing and manipulating \iterm{objects}(See~\reffig{fig:scenario_objects}).
-The TC will compile a list of at least \NumObjects objects for this purpose, assigning them official names.
-Most objects are likely to be lightweight and easy to grasp with one hand.
-Each object has assigned a category (e.g. an \textit{apple} and a \textit{banana} belong to the \textit{fruits} category).
-Each \iterm{object category} has assigned a \iterm{predefined location} (e.g. an \textit{fruits} can be found in the \textit{kitchen table}).
-Assignments are announced during setup days (See~\refsec{chap:setup_and_preparation}).
-An exemplar of each object is provided before the competition for training.
+\label{sec:rules:scenario:objects}
+Some tests involve recognizing and manipulating objects (See~\reffig{fig:scenario_objects}).
+The \TC{} will compile a list of at least \NumObjects{} objects at the competition. This list contains a picture of the object, as well as, its official name and \ObjectCategory{}. Every \ObjectCategory{} has an assigned \PredefinedLocation{} (see \ref{sec:rules:scenario:locations}) where objects of that category can usually be found during tests.
+Each object is provided at the competition for training.
 
 There are two types of objects:
 
 \begin{enumerate}
-	\item \textbf{\iterm{Known objects}:} Objects previously known by the robot and that it can identify and manipulate.
-	There are two kinds of known objects:
+	\item \textbf{\KnownObjects{}:} Objects previously known by the robot, split into:
 	\begin{enumerate}
-		\item \textbf{\iterm{Regular objects}:} Objects with no noticeable difference among peers (e.g.~soda can, cereal box, cutlery, etc).
-		\item \textbf{\iterm{Alike objects}:} Objects which are different one from another, but still considered by people to be the same (e.g.~apple, sandwich, cloth, etc.).
+		\item \textbf{\RegularObjects{}:} Objects that always look the same.
+		\item \textbf{\AlikeObjects{}:} Objects that look different to the provided image but are still considered the same by people (e.g. differently colored apple, cloth with different pattern).
+		\item \textbf{\StandardObjects{}:} Objects chosen from the \YCBData{}\footnote{\url{http://www.ycbbenchmarks.com/object-set/}}. They are published 6 months in advance on the \RoboCup\AtHome{} website\footnote{\url{https://athome.robocup.org/standard-objects}} so that they can be aquired and trained beforehand.
```

Comment:
Instead of the other two categories, would have "non-standard" and say that they are purchased in the country and made available during setup days (reference to set up days section).

Reply:
As I said, imo regular and alike is an important distinction.